readme.md

```diff
@@ -1076,7 +1089,15 @@ A **product** type combines types together in a way you're probably more familia
 // point :: (Number, Number) -> {x: Number, y: Number}
 const point = (x, y) => ({ x, y })
 ```
-It's called a product because the total possible values of the data structure is the product of the different values. Many languages have a tuple type which is the simplest formulation of a product type.
+It's called a product because the total possible values of the data structure is the product of the different values.
+
+```ts
+type Rank = 'A' | '2' | '3' | '4' | '5' | '6' | '7' | '8' | '9' | '10' | 'J' | 'Q' | 'K'
```

Comment:
Any other alternatives? @jethrolarson 🤓

Reply:
Infinitely many, but why?

Reply:
I figured playing cards were something that a lot of people are familiar with whose instances are a product of two sets.

Reply:
I meant instead of using `ts` :)

Reply:
Oh, it's a concept that only exists in type space so some kind of type syntax is necessary to describe it. Hmm I don't know how this would be represented in the pseudo-type system this document uses.

In F# which is a hindley-milner system it'd be something like 
```
type Rank = A | Two | Three  | Four | Five | Six | Seven | Eight | Nine | Ten | J | Q | K
type Suit = Spades | Hearts | Clubs | Diamonds
type Card = Rank * Suit
```
Is that better?

Reply:
hindley-milner for consistence, sure! Thanks.

Reply:
My reticence is then that these named types have no representation in value-space and thus may make no sense in JS.

Reply:
nods, but hindley-milner would be bit more consistent?